contents/blog/mixpanel-alternatives.md

```diff
@@ -0,0 +1,322 @@
+---
+title: "The most popular Mixpanel alternatives, compared"
+date: 2023-09-26
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["andy-vandervell"]
+featuredImage: ../images/blog/posthog-alternatives/posthog-alternatives.jpeg
+featuredImageType: full
+category: General
+tags:
+  - Comparisons
+---
+
+
+> ### TLDR
+>
+> - **PostHog:**
+
+
+## PostHog
+
+![posthog](../images/blog/mixpanel-alternatives/posthog-website.png)
+
+### What is PostHog?
+
+PostHog is an all-in-one platform that combines product analytics, session replay, feature flags, A/B testing, and surveys into one product. It can also replace customer data platforms (CDP), like Segment.
+ 
+### Key features
+
+- 📈 **Product analytics**, including funnel insights, user paths, retention analysis, custom trends, and dynamic user cohorts. Power users can create custom insights using SQL.
+
+- 📺 **Session replays** with event timelines, console logs, and network activity. You can also download recordings to save them beyond the standard 90-day retention policy.
+
+- 🚩 **Feature flags** with local evaluation (for faster performance) and JSON payloads, so you can make changes to your product without deploying new code. 
+
+- 🧪 **A/B tests** with up to 9 test variations, primary and secondary metrics. Automatically calculates recommended duration, sample size, and statistical significance.
+
+- 💬 **Surveys**, including open text, link with custom CTA, ratings (for NPS and PMF surveys), single and multiple choice. Target users based on user properties, URL, or using existing feature flags.
+
+### Who uses PostHog?
+
+Typical PostHog users are:
+
+- Product-minded engineering teams
+- Data-savvy product managers
+- Startups and mid-size enterprises
+
+Customers include [AssemblyAI](/customers/assemblyai), [Hasura](/customers/hasura), [Vendasta](/customers/vendasta), and Airbus.
+
+### How does PostHog compare to Mixpanel?
+
+PostHog's built-in features make it the obvious choice if you like the idea of consolidating around one platform. Mixpanel's offers plentiful third-party integrations for features it doesn't have, but this creates a messy workflow for teams.
```

Comment:
```suggestion
PostHog's built-in features make it the obvious choice if you like the idea of consolidating around one platform, and it can be extended further thanks to it being open source. Mixpanel offers plentiful third-party integrations for features it doesn't have, but this creates a messy workflow for teams. 
```